### PR TITLE
add parameter n

### LINKED ISSUE #10: Parameter -n

### PR DESCRIPTION
**Pull request by justinmueller05082000**
fixes #10

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -28,6 +28,7 @@ var regArr []*regexp.Regexp
 var nInvertParameter = flag.Bool("v", false, "use PATTERN as non-matching lines.")
 var quietParameter = flag.Bool("q", false, "suppress all normal output")
 var maxCountParameter = flag.Int("m", -1, "stop after NUM selected lines.")
+var lineNumberParameter = flag.Bool("n", false, "print line number before file name and matches.")
```
Comment (Al2Klimov) [inline main.go]: "before file name"?

**Review by Al2Klimov — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (justinmueller05082000) [reply main.go]: i noticed it later but forgot to change it. Would "print line number with output" be enough?

Comment (Al2Klimov) [reply main.go]: "print" implies "output", so that's even too long.

Comment (justinmueller05082000) [reply main.go]: so just "print line number"?

Comment (Al2Klimov) [reply main.go]: Yes.

**Review by Al2Klimov — CHANGES_REQUESTED** [review]
* [x] `git reset --soft HEAD~$(git rev-list --count $(git merge-base HEAD master)..HEAD^)`
* [x] `git commit --amend --no-edit`
* [x] `git push -f origin feature/add-parameter-n-10`

**Review by Al2Klimov — CHANGES_REQUESTED** [review]
Write a test protocol w/ and w/o -n.

**Comment by justinmueller05082000** [comment]
**Test protocol:**

**toGrep.txt:**
```
b
a
a
a
a
```

**Case 1 (w/ -n):**
**Terminal:** `go run . -n -e 'a' < toGrep.txt`
**Output:**
```
2:a
3:a
4:a
5:a
```

**Terminal:** `grep -n -e 'a' < toGrep.txt `
**Output:**
```
2:a
3:a
4:a
5:a
```
___________________

**Case 2 (w/o -n):**
**Terminal:** `go run . -e 'a' < toGrep.txt`
**Output:**
```
a
a
a
a
```

**Terminal:** `grep -e 'a' < toGrep.txt`
**Output:**
```
a
a
a
a
```
___________________

**Conclusion:**
All outputs are equal except for the colored highlighting in the terminal.

**Review by Al2Klimov — APPROVED** [review]

**Pull request merged**